### PR TITLE
Adding users db model and tests

### PR DESCRIPTION
**Pull request by kmiddleton14**

### CODE REVIEW

File: db/models/city.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
```
Comment (tessaSAC) [inline db/models/city.js:4]: Why is the APP folder all caps? Jc

File: db/models/test/city.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const db = require('APP/db')
+const City = require('APP/db/models/city')
+const {expect} = require('chai')
+
+describe('City', () => {
+  before('wait for the db', () => db.didSync)
```
Comment (tessaSAC) [inline db/models/test/city.test.js:8]: Do you need some kind of corollary `after` function that cleans up the db?

File: db/models/order.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Order = db.define('order', {
+  totalPrice: {
+    type: Sequelize.DECIMAL(10, 2),
```
Comment (tessaSAC) [inline db/models/order.js:8]: Why DECIMAL?

File: db/models/city.js (hunk 1 above)
Comment (kmiddleton14) [reply db/models/city.js:4]: This is how bones was structured and given to us in the template

**Comment by csjiang** [comment]
Don't merge into a branch from master; merge into master from a base branch- Galen

**Pull request closed without merging**